app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
```

Comment:
- [ ] Please know that the application should ask for confirmation of the email, you can do that by adding `:confirmable` here, and adjusting another setting I'll mention below.

Reply:
- [ ] This should be addressed as suggested with the previous reviewer